docker-hub/repos/configure/index.md

```diff
@@ -3,55 +3,47 @@ description: Configuring repositories on Docker Hub
+You can make an existing repository private by going to its **Settings** tab and select **Make private**.
 
-![Convert Repo to Private](../../images/repo-make-private.png){: style="max-width: 60%"}
+![Repo make private](../../images/repo-make-private.png){: style="max-width: 60%"}
 
-You get one private repository for free with your Docker Hub user account (not
-usable for organizations you're a member of). If you need more private
-repositories for your user account, upgrade your Docker Hub plan from your
-[Billing Information](https://hub.docker.com/billing/plan){: target="_blank" rel="noopener" class="_"} page.
+You get one free private repository with your Docker Hub user account (not
+available for organizations you're a member of). If you need more private
+repositories for your user account, upgrade your Docker Hub subscription from your [Billing Information](https://hub.docker.com/billing/plan){: target="_blank" rel="noopener" class="_"} page.
 
-Once you create the private repository, you can `push` and `pull` images to and
+Once you've create a private repository, you can `push` and `pull` images to and
 from it using Docker.
 
 > **Note**: You must sign in and have access to work with a
-> private repository.
-
-> **Note**: Private repositories aren't available to search through
+> private repository. Private repositories aren't available to search through
 > the top-level search or `docker search`.
 
 You can choose collaborators and manage their access to a private
 repository from that repository's **Settings** page. You can also toggle the
 repository's status between public and private, if you have an available
 repository slot open. Otherwise, you can upgrade your
-[Docker Hub](https://hub.docker.com/account/billing-plans/){: target="_blank" rel="noopener" class="_"} plan.
+[Docker Hub](https://hub.docker.com/account/billing-plans/){: target="_blank" rel="noopener" class="_"} subscription.
 
 ### Permissions reference
 
 Permissions are cumulative. For example, if you have Read & Write permissions,
 you automatically have Read-only permissions:
 
-- `Read-only` access allows users to view, search, and pull a private repository in the same way as they can a public repository.
-- `Read & Write` access allows users to pull, push, and view a repository Docker
-  Hub. In addition, it allows users to view, cancel, retry or trigger builds
-- `Admin` access allows users to Pull, push, view, edit, and delete a
-  repository; edit build settings; update the repository description modify the
-  repositories "Description", "Collaborators" rights, "Public/Private"
-  visibility, and "Delete".
+- `Read-only` access lets users view, search, and pull a private repository in the same way as they can a public repository.
+- `Read & Write` access lets users pull, push, and view a repository Docker
+  Hub. In addition, it lets users view, cancel, retry or trigger builds
```

Comment:
Continued from the previous line if removing `Docker Hub`.

```suggestion
  . In addition, it lets users view, cancel, retry, or trigger builds.
```

Reply:
Updated